jni_gen/third_party/ApiSummarizer/src/main/java/com/github/hegde/mahesh/apisummarizer/doclet/ElementBuilders.java

```diff
@@ -0,0 +1,215 @@
+package com.github.hegde.mahesh.apisummarizer.doclet;
+
+import com.github.hegde.mahesh.apisummarizer.elements.*;
+import com.github.hegde.mahesh.apisummarizer.util.StreamUtil;
+import com.sun.source.doctree.DocCommentTree;
+import java.util.HashMap;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import javax.lang.model.element.*;
+import javax.lang.model.type.*;
+
+public class ElementBuilders {
+  AstEnv env;
+
+  private static <T, R> List<R> collect(List<T> elements, Function<T, R> mapper) {
```

Comment:
The method collect(List<T>, Function<T,R>) from the type ElementBuilders is never used locally.

Maybe run a Java analyzer over the codebase.
Can we also run a Java analyzer on the CI?

Here's what my IDE is reporting: https://gist.github.com/dcharkes/77f3ec4c99f4a4f63faed385a1817c60

Reply:
Correct. At some point replaced it with `StreamUtils.mapped`.